docs/api/download-item.md

```diff
@@ -88,6 +88,19 @@ Returns `String` - The save path of the download item. This will be either the p
 set via `downloadItem.setSavePath(path)` or the path selected from the shown
 save dialog.
 
+#### `downloadItem.setSaveDialogOptions(options)`
+
+* `options` Object - Set the save file dialog options. This object has the same
+properties as the `options` parameter of [`dialog.showSaveDialog()`](dialog.md).
+
+This API allows the user to set custom options for the save dialog that opens
+for the download item by default.
+The API is only available in session's `will-download` callback function.
+
+#### `downloadItem.getSaveDialogOptions()`
+
+Returns `Object` - Returns the object previously set by `downloadItem.setSaveDialogOptions(options)`.
```

Comment:
Does this return the defaults if they haven't been previously set?

Reply:
The defaults are only set after the JS callback actually runs AFAIK, so nope.